okhttp-apache/src/test/java/com/squareup/okhttp/apache/OkApacheClientTest.java

```diff
@@ -113,6 +117,16 @@
     assertEquals("Hello, world!", request.getBody().readUtf8());
     assertEquals(request.getHeader("Content-Length"), "13");
   }
+  @Test public void postEmptyEntity() throws Exception {
+    server.enqueue(new MockResponse());
+    final HttpPost post = new HttpPost(server.getUrl("/").toURI());
+    client.execute(post);
```

Comment:
These tests don't assert anything. Copy the assertions from another test and ensure the body was indeed missing.